account_payment_term_extension/tests/test_account_payment_term_multi_day.py

```diff
@@ -112,7 +108,7 @@ def setUpClass(cls):
                         {
                             "value": "percent_amount_untaxed",
                             "value_amount": 10.0,
-                            "option": "day_after_invoice_date",
+                            # "option": "day_after_invoice_date",
```

Comment:
no need to keep if field has been removed

Reply:
> Hi Hoang,
> 
> * I believe the `compute()` method has been fully removed in favor of `_compute_terms()` in native so it should probably be removed from `account_payment_terms.py`
> * in the description of your PR, you might want to make explicit that a few use cases are now supported natively
> * otherwise lgtm!

yep, you're right, my bad. I will remove redundant code